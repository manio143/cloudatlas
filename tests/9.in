SELECT land(cpu_usage < 0.5) AS cpu_ok
