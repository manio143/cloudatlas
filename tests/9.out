/uw: cpu_ok: false
/pjwstk: cpu_ok: true
